source/reporting/index.html.md.erb

```diff
@@ -58,48 +58,155 @@ You can use the GOV.UK Pay API to:
 
 `GET /v1/payments/{paymentId}`
 
-See the <a href="https://govukpay-api-browser.cloudapps.digital/#getpayment" target="blank">GOV.UK Pay API browser</a> for more details
+Example response:
 
-If the payment exists, the JSON response body to this
-API call contains a `state` field. You can use this information when
-designing a service.
+```json
+{
+  "created_date": "2019-07-11T10:36:26.988Z",
+  "amount": 4000,
+  "state": {
+    "status": "success",
+    "finished": true,
+  },
+  "description": "Pay your council tax",
+  "reference": "12345",
+  "language": "en",
+  "metadata": { 
+    "ledger_code": "AB100",
+    "an_internal_reference_number": 200
+  },
+  "email": "sherlock.holmes@example.com",
+  "card_details": {
+    "card_brand": "Visa",
+    "card_type": "debit",
+    "last_digits_card_number": "1234",
+    "first_digits_card_number": "123456",
+    "expiry_date": "04/24",
+    "cardholder_name": "Sherlock Holmes",
+    "billing_address": {
+        "line1": "221 Baker Street",
+        "line2": "Flat b",
+        "postcode": "NW1 6XE",
+        "city": "London",
+        "country": "GB"
+    }
+  },  
+  "payment_id": "hu20sqlact5260q2nanm0q8u93",
+  "refund_summary": {
+    "status": "available",
+    "amount_available": 4000,
+    "amount_submitted": 0
+  },
+  "settlement_summary": {
+    "capture_submit_time": "2019-07-12T17:15:000Z",
+    "captured_date": "2019-07-12",
+    "settled_date": "2019-07-12"
+  },
+  "delayed_capture": false,
+  "moto": false,
+  "corporate_card_surcharge": 250,
+  "total_amount": 4000,
+  "fee": 200,
+  "net_amount": 3800,
+  "payment_provider": "worldpay",
+  "provider_id": "10987654321",
+  "return_url": "https://your.service.gov.uk/completed"
+}
+```
 
-For example, the response for a successful payment contains:
+You know that your user can continue their journey
+through your service if in the `state` object the:
```

Comment:
For some reason I found this sentence a bit difficult to comprehend and had to read it a few times, although it does make sense.

Reply:
Yes I agree! I was never quite happy with this. I'll tweak and add some commas to make it a bit easier to read.